crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -175,6 +182,110 @@ impl PgManager {
             .await
     }
 
+    async fn multi_get_coins(
+        &self,
+        address: Vec<u8>,
+        coin_type: Option<String>,
+        first: Option<u64>,
+        after: Option<String>,
+        last: Option<u64>,
+        before: Option<String>,
+    ) -> Result<Option<(Vec<StoredObject>, bool)>, Error> {
+        let mut query = objects::dsl::objects.into_boxed();
+        query = query
+            .filter(objects::dsl::owner_id.eq(address))
+            .filter(objects::dsl::owner_type.eq(1));
```

Comment:
Would it be useful to a comment to make it easier to remember what owner_type 1 is?

Reply:
It's here so we can leverage the index on objects table, I had considered adding a comment but not sure if we should or not

Reply:
ok, I leave it at your discretion!